zerver/webhooks/yo/doc.md

```diff
@@ -0,0 +1,28 @@
+See your Yo App notifications in Zulip!
+
+First, on your {{ settings_html|safe }}, create a bot for
```

Comment:
Since email is the core parameter here, I felt using the create-bot-construct-url template would be a bad fit, even if we fix the "First" vs "Next" issue.